openj9.build/include/top.xml

```diff
@@ -510,6 +510,16 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 					<contains string="@{arch}" substring="arm"/>
 				</and>
 			</condition>
+			<condition property="@{property}" value="aarch32">
+				<and>
+					<contains string="@{arch}" substring="aarch32"/>
+				</and>
+			</condition>
+			<condition property="@{property}" value="aarch64">
+				<and>
+					<contains string="@{arch}" substring="aarch64"/>
+				</and>
+			</condition>
```

Comment:
Could we be more generic? For example, set `property` to `@{arch}` be the default and only handle exception cases?
In the current code, we will have to update it whenever there is a new platform introduced.

Reply:
The task of generalizing the logic is not very straight forward, as the code "Sets a property containing a normalised architecture string from the java os.arch property"; i.e., in the code we are doing a "contains" search in `@{arch}` for a normalized architecture string, e.g. "ppc", and if found, we are setting the property to that normalized string. If we set `property` to `@{arch}` directly, then we won't be setting it to the normalized name of the architecture, but the full name -- which violates the original purpose of the macrodef. However, in any case, if we do want to find a generic solution to avoid having to add the architecture name each time we have a new one, we should ideally handle that in a separate PR.

Reply:
If `arch` is not the exact value that we want, could we use `PLATFORM` or `SPEC` and avoid to construct the value again in system tests? `PLATFORM` and `SPEC` should be defined before or detected at the beginning of running TKG.

But I agree, we can handle it in another PR.

Reply:
We could. I'll open a separate PR to investigate this further. Thanks Lan!